Realm/Tests/SwiftUISyncTestHostUITests/SwiftUISyncTestHostUITests.swift

```diff
@@ -548,3 +557,134 @@ extension SwiftUISyncTestHostUITests {
         XCTAssertTrue(waitingUserView.waitForExistence(timeout: 2))
     }
 }
+
+class SwiftUIFlexibleSyncTestHostUITests: SwiftUISyncTestCases {
+    override func setUp() {
+        continueAfterFailure = false
+
+        try? FileManager.default.createDirectory(at: clientDataRoot, withIntermediateDirectories: true)
+        try? FileManager.default.createDirectory(at: appClientDataRoot, withIntermediateDirectories: true)
+
+        // Create App once for this Test Suite
+        if SwiftUISyncTestHostUITests.flexibleSyncAppId == nil {
+            do {
+                let appId = try RealmServer.shared.createAppWithQueryableFields(["age", "firstName"])
+                SwiftUISyncTestHostUITests.flexibleSyncAppId = appId
+            } catch {
+                XCTFail("Cannot initialise test without a creating an App on the server")
```

Comment:
This error doesn't make much sense to me? It's also discarding the actual error message.